modules/openxr/openxr_api.cpp

```diff
@@ -2447,6 +2478,21 @@ void OpenXRAPI::end_frame() {
 		render_state.view_count, // viewCount
 		render_state.projection_views, // views
 	};
+
+	// @todo Is this an OK time to do this for multithreading?
```

Comment:
Let's remove this TODO comment. I think it is OK (because this function will run on the render thread), but if it isn't, it's something we should handle now and not later :-)